src/common/snippets/src/lowered/loop_manager.cpp

```diff
@@ -438,6 +463,31 @@ void LinearIR::LoopManager::fuse_loops(LinearIR::constExprIt loop_begin_target,
     remove_loop_info(from);
 }
 
+std::vector<lowered::pass::PassPipeline> LinearIR::LoopManager::fuse_loop_handlers(
+    std::vector<lowered::pass::PassPipeline>& from,
+    std::vector<lowered::pass::PassPipeline>& to) {
+    const auto min_size = std::min(from.size(), to.size());
```

Comment:
Could you tell me please in which cases the count of pipelines might be different?

Reply:
I suppose such situations are theoretically possible in dynamic scenario, and exactly in tail processing part: a loop can have several precompiled tail bodies with different tail size for more effective execution (e.g. `tail_size_0=1`, `tail_size_1=4`, `tail_size_2=8`). So last loop iteration with `work_amount = 13` will be processed via 3 bodies: `8 (tail_size_2) + 4 (tail_size_1) + 1 (tail_size_0)`.

Theoretically, 2 fused loops can have different sets of precompiled bodies for processing the last iteration - in this case count of pipelines will be different.

However, I am not sure that we will meet this theoretical situation in the reality, so maybe it is better to prevent loops fusion in case of difference in pipelines count, and remove handling of such cases from `fuse_loop_handlers`. What do you think?

Reply:
Discussed offline: since the described situation can't be meet in real scenarios for now and it's not clear how exactly the described mechanism will be implemented, it was decided to remove a specific handling for such situations from this PR.